frontend/src/components/QuestionInput/AlertContainer.tsx

```diff
@@ -15,7 +15,9 @@ export const AlertContainer = ({ onClose, alerts }: AlertContainerProps) => {
       className={`alertContainer display-flex flex-column width-full  ${styles.errorAlertContainer}`}
     >
       {alerts.map((alert) => (
-        <AlertBanner onClose={onClose} alert={alert} />
+        <div key={alert.id}>
+          <AlertBanner onClose={onClose} alert={alert} />
+        </div>
```

Comment:
You actually don't need the wrapper element when adding a `key` prop to a component. `key` is always a valid prop that you don't need to add to your prop types. https://legacy.reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys
```suggestion
        <AlertBanner onClose={onClose} key={alert.id} alert={alert} />
```